app/views/cookies/show.html.erb

```diff
@@ -1,13 +1,9 @@
-
 <div id="content">
-
   <h1>Privacy, cookies, and third party services</h1>
 
-  <h2>Cookies</h2>
-
   <p>To make our service easier or more useful, we sometimes place small data files on your computer or mobile phone, known as cookies; many websites do this. We use this information to, for example, prevent duplicate votes, or to measure how people use the website so we can improve it and make sure it works properly. Below, we list the cookies and services that this site can use.</p>
 
-  <h3>Strictly necessary cookies</h3>
+  <h2>Strictly necessary cookies</h2>
```

Comment:
Shouldn't these be `<h3>`s semantically?

Reply:
I thought the principle was to not have "gaps", so why would jumping from `h1` to `h3` be better?

Reply:
Oh, sorry, I misread. Yeah, that's fine 👍